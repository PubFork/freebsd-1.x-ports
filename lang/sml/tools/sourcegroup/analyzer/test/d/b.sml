local structure A = struct val x = B.y end in
  structure H = struct val y = A.x end
end
