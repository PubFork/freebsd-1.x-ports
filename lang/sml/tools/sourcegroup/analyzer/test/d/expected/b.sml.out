source sml "d/b.sml"
  import structure B 
  export structure H ;

